src/SAWScript/Builtins.hs

```diff
@@ -468,12 +474,49 @@ printGoalSize =
      printOutLnTop Info $ "Goal unshared size: " ++ show (scTreeSize t)
      return ((), mempty, Just goal)
 
+resolveNames :: SharedContext -> [String] -> TopLevel (Set VarIndex)
+resolveNames sc nms = Set.fromList <$> mapM (resolveName sc) nms
+
+-- | Given a user-provided name, resolve it to some
+--   'ExtCns' that represents an unfoldable 'Constant'
+--   value or a fresh uninterpreted constant.
+--
+--   We first attempt to find the name in the local Cryptol
+--   environment; if the name is found, attempt to resolve it to
+--   an 'ExtCns' in the SAWCore environment.  If the given name
+--   does not resolve to a cryptol value in the current environment that
+--  maps to an 'ExtCns', then instead directly look it up
+--  in the SAWCore naming environment.  If both stages
+--  fail, then throw an exception.
+resolveName :: SharedContext -> String -> TopLevel VarIndex
+resolveName sc nm =
```

Comment:
how does the error message look like when the string is not a valid name?

Reply:
You can see them here https://github.com/GaloisInc/saw-core/blob/94c8e51c509cb3379fec3a8710c442b8a2b6ae58/saw-core/src/Verifier/SAW/SharedTerm.hs#L408

You'll get a message that looks like:

```
Could not resolve name: "asdf"
```

or 

```
Ambiguous name "asdf" might refer to any of the following:
Module1::asdf
Module2::asdf
```

Reply:
I mean, how does the whole error message look like, as printed by saw. Does it contain location information?

Reply:
```
$ cat test.saw

unfold_term ["asdf"] {{ 42 }};

$ saw test.saw

[23:14:23.327] Loading file "/Users/rdockins/code/saw-script/test.saw"
[23:14:23.343] Stack trace:
"unfold_term" (/Users/rdockins/code/saw-script/test.saw:2:1-2:12):
Could not resolve name: "asdf"
```

Reply:
that's what I was looking for, thanks!